### ISSUE TITLE
Not render without container with fixed height

### ISSUE THREAD

**Issue by bernex** [fritz-c/react-sortable-tree#33]
I wanna render in page without fixed height and use main page scrolling.

```
<div style={{ height: 50 }}>
                    <SortableTree
                        treeData={treeData}
                        onChange={this.updateTreeData}
                        onMoveNode={({ node, treeIndex, path }) =>
                                console.debug( // eslint-disable-line no-console
                                    'node:', node,
                                    'treeIndex:', treeIndex,
                                    'path:', path,
                                )
                            }
                        rowHeight={20}
                        maxDepth={2}
                    />
                </div>
```

If i remove `<div style={{ height: 50 }}>` -> nothing visible.
Why?

**Comment by fritz-c** [comment]
Please refer to the discussion in https://github.com/fritz-c/react-sortable-tree/issues/15

**Comment by fritz-c** [comment]
I got this request enough that I implemented a fix in v0.1.9. By setting the `isVirtualized` prop to `false`, you can draw the tree without `react-virtualized`.

One thing to note, though. The following functionality worked under the assumption that the tree container would be fixed-height and scrollable, and therefore doesn't work without virtualization:
- Auto-scrolling while dragging
- Scrolling to the `searchFocusOffset`
